src/ResourceManager/EventHub/Commands.EventHub/help/Get-AzureRmEventHubAuthorizationRule.md

```diff
@@ -122,16 +163,15 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
-### CommonParameters
```

Comment:
This shouldn't have been removed - please regenerate this file.

Reply:
Resolved

Reply:
Can you update to the latest version of platyps and regenerate again? Every cmdlet should have the common parameters.